os/iphone-latest-sdk.cmake

```diff
@@ -0,0 +1,16 @@
+# This script sets IOS_SDK_VERSION with the version number 
+# of the latest iPhoneSDK found on the disk (for example 11.0) 
+
+set(_POLLY_IOS_SDK_ROOT "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs")
```

Comment:
We can't rely on this hardcoded path. User is free to use any directory:
* http://cgold.readthedocs.io/en/latest/first-step/native-build-tool/xcode.html#several-custom-xcode-versions